search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/filter/DecisionInstanceFilterTransformer.java

```diff
@@ -96,12 +81,13 @@ private SearchQuery getProcessInstanceKeysQuery(final List<Long> processInstance
     return longTerms("processInstanceKey", processInstanceKeys);
   }
 
-  private SearchQuery getDecisionDefinitionKeysQuery(final List<Long> decisionDefinitionKeys) {
-    return stringTerms(
-        "decisionDefinitionId",
-        decisionDefinitionKeys != null
-            ? decisionDefinitionKeys.stream().map(String::valueOf).toList()
-            : null);
+  private List<SearchQuery> getDecisionDefinitionKeysQuery(
+      final List<Operation<Long>> decisionDefinitionKeyOperations) {
+    final var stringOperations =
+        decisionDefinitionKeyOperations.stream()
+            .map(op -> new Operation<>(op.operator(), String.valueOf(op.value())))
+            .collect(Collectors.toList());
```

Comment:
🙃 You can abbreviate this to `.toList()` (since Java 9 or 11 or something)
```suggestion
            .toList();
```